client/lib/domains/cart-utils.js

```diff
@@ -13,7 +13,7 @@ import { isDomainRegistration } from 'calypso/lib/products-values/is-domain-regi
  * Depending on the current step in checkout, the user's domain can be found in
  * either the cart or the receipt.
  *
- * @param {?object} receipt - The receipt for the transaction
+ * @param {?object} [receipt] - The receipt for the transaction
```

Comment:
This is peripheral, but I'm a little confused about why `receipt` is an array here.

Reply:
Heh, that's the wrong question. I'm more confused than I thought. What do the square brackets mean here?

Reply:
They are how you say "this param is optional" in jsdoc. 🤷